packages/block-editor/src/components/block-preview/index.js

```diff
@@ -29,10 +32,14 @@ export function BlockPreview( {
 	}
 	return (
 		<BlockEditorProvider value={ renderedBlocks } settings={ settings }>
-			<AutoHeightBlockPreview
-				viewportWidth={ viewportWidth }
-				__experimentalPadding={ __experimentalPadding }
-			/>
+			{ __experimentalLive ? (
```

Comment:
Is the difference here just to avoid the "scaling" ? Maybe we can have a "enbaleScaling" prop? "live" is a bit confusing.

Reply:
The scaling, block memoization, and other nuance.